core/libs/aspect-exporter/src/lib/visitor/entity-value/entity-value-visitor.ts

```diff
@@ -42,22 +40,60 @@ export class EntityValueVisitor extends BaseVisitor<DefaultEntityValue> {
     const {aspectModelUrn} = entityValue;
     const rdfModel = this.modelService.currentRdfModel;
 
-    entityValue.properties.forEach(property => {
-      const object = this.createObjectForRDF(property);
-      const {key, value} = property;
+    const {propertyCollectionWithLangString, property} = entityValue.properties.reduce(
+      (acc, property) => (
+        isDataTypeLangString(property.key.property) && property.key.property.characteristic instanceof DefaultCollection
```

Comment:
Is it working also for Set, SortedSet and all other list definition?

Reply:
done

Reply:
yes, it is working